src/content/docs/ko/guides/backend/sentry.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Sentry로 Astro 사이트를 모니터링하세요
+description: Sentry로 Astro 사이트를 모니터링하는 방법
+type: backend
+service: Sentry
+stub: false
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+[Sentry](https://sentry.io)는 개발자가 실시간으로 문제를 식별, 진단 및 해결할 수 있도록 설계된 포괄적인 애플리케이션 모니터링 및 오류 추적 서비스를 제공합니다.
+
+[Sentry와 Astro의 파트너십](https://astro.build/blog/sentry-official-monitoring-partner/) 및 Astro 개발 환경에 풍부한 디버그 오버레이를 제공하는 Sentry의 Spotlight 개발 도구 모음 앱에 대해 블로그에서 자세히 알아보세요. Spotlight는 로컬 개발 중 발생하는 오류, 추적 및 중요한 컨텍스트를 브라우저에 바로 표시합니다.
+
+Sentry의 Astro SDK를 사용하면 Astro 애플리케이션의 오류 및 추적 데이터를 자동으로 보고할 수 있습니다.
+
+## 프로젝트 구성
+
+필수 구성 요소의 전체 목록은 [Astro용 Sentry 가이드](https://docs.sentry.io/platforms/javascript/guides/astro/#prerequisites)에서 확인할 수 있습니다.
+
+## 설치
+
+Sentry는 애플리케이션 런타임에서 SDK를 사용하여 데이터를 캡처합니다.
+
+Astro CLI에서 선택한 패키지 관리자에 대해 다음 명령을 실행하여 SDK를 설치합니다.
+
+```bash
+npx astro add @sentry/astro
+```
```

Comment:
We just need to add your changes from https://github.com/withastro/docs/commit/6125b43ddefd43e44f0b2d515f21bcff66a1d794 and then LGTM

Reply:
The update has been completed!